src/utils/getApplicableFormats.ts

```diff
@@ -3,6 +3,14 @@ import { RegularNode, SchemaNodeKind } from '@stoplight/json-schema-tree';
 import { COMMON_JSON_SCHEMA_AND_OAS_FORMATS } from '../consts';
 
 export function getApplicableFormats(schemaNode: RegularNode): [type: SchemaNodeKind, format: string] | null {
+  // Format is handled for contentMediaType type i.e. binary format
```

Comment:
Please write comments that describe *why* your code does what it does, rather than *what* it does.  I think this comment can be removed.

Reply:
Okay got it. I'll remove this and will add another comment describing why we are doing this.